ipn/localapi/localapi.go

```diff
@@ -919,6 +919,11 @@ func (h *Handler) serveDebugPortmap(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// EventError is a simple struct for wrapping errors
```

Comment:
Minor nit, I like to avoid self-describing things as simple
```suggestion
// EventError provides the JSON encoding of internal errors from event processing.
```

or words to that effect.